### PR TITLE
Css canvas medical - YoonHwan Cho

### PR DESCRIPTION
**Pull request by YoonHCho**
Css canvas medical - YoonHwan Cho
CSS Canvas Medical

### CODE REVIEW

**Comment by YoonHCho** [comment]
Completed practice exercise css canvas product
CSS Canvas Product

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Yoon, your fonts are wrong in multiple different places, I see uses of an incorrect font family, wrong font sizes, and possibly some incorrect font-weight and font-style. Please revise this and submit a new screenshot with your revision.

**Comment by YoonHCho** [comment]
removed the address element and the italic from it. changed the font style on the headers and sizing on some
CSS Canvas Medical

**Comment by YoonHCho** [comment]
removed the address element and the italic from it. changed the font style on the headers and sizing on some
2022-07-06 13_02_00-JavaScript Data Model

File: css-canvas-medical/index.html [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+          <li class="nav-inline justify-right indent-nav">
+            <button class="btn-contact font-weight-100">Contact Us</button>
+          </li>
+        </ul>
+      </section>
+    </header>
+
+    <hr>
+
+    <div class="spacing"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:34]: You shouldn't use empty elements for spacing, you should be using `margin` or `padding`

File: css-canvas-medical/index.html [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+      <div class="fade"></div>
+      <div class="column-three-fifth margin-4rem absolute">
+        <h1 class="font-white font-t padding-right-13rem font-weight-400 font-size-2p2rem">The EMR Reimagined for Primary Care</h1>
+        <p class="opacity font-white font-a line-height-2 padding-right-9rem font-size-0p97">Can't remember the last time you talked with a patient without focusing on the EMR? Canvas lets you rediscover joy in the work you do. Capture meaningful data, identify gaps in care, and move seamlessly between tasks within the platform. You stay present with patients while your whole care team stays up-to-speed. Be a better doctor with Canvas, designed for primary care providers with the help of primary care providers.</p>
+        <div>
+          <button class="btn-started font-a font-size-1rem font-weight-100">Get Started</button>
+        </div>
+      </div>
+    </section>
+    <div class="spacing"></div>
```
Comment (codingcodymiller) [inline css-canvas-medical/index.html:45]: You shouldn't use empty elements for spacing, you should be using `margin` or `padding`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by YoonHCho** [comment]
Hi Cody, removed the spacing and used the margin-top and bottom
CSS Canvas Medical

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Comment by YoonHCho** [comment]
completed canvas practice product.html & product.css
CSS Canvas product

**Comment by YoonHCho** [comment]
completed canvas practice implementation.html & implementation.css

CSS Canvas implementation

**Pull request merged**